core/domain/summary_services.py

```diff
@@ -256,7 +260,8 @@ def get_exploration_metadata_dicts(exploration_ids, user):
 
     filtered_exploration_summaries = []
     for (exploration_summary, exploration_rights) in (
-            zip(exploration_summaries, exploration_rights_objects)):
+            list(python_utils.ZIP(
```

Comment:
No need for list here, just use ZIP directly.

Reply:
Done